build-system/global-configs/experiments-const.json

```diff
@@ -1,4 +1,5 @@
 {
-  "INTERSECTION_OBSERVER_POLYFILL": true,
+  "INTERSECTION_OBSERVER_POLYFILL": false,
```

Comment:
/to @rsimha @kevinkimball we need to discuss this diff. Is this just an accident or some tooling problem? If tooling, this could be a significant issue. In effect this launched an experiment that was still in the trial phase (and unlaunched an already launched one). Fortunately, the trial looks good and this is benign, but could be more traumatic.

Reply:
I was under the impression this was an intentional, manual change. @kevinkimball can correct me if I'm wrong.